crypto/asn1/a_time.c

```diff
@@ -468,6 +469,11 @@ static const char _asn1_mon[12][4] = {
 };
 
 int ASN1_TIME_print(BIO *bp, const ASN1_TIME *tm)
+{
+    return ASN1_TIME_print_ex(bp, tm, 0);
+}
+
+int ASN1_TIME_print_ex(BIO *bp, const ASN1_TIME *tm, int also_invalid)
```

Comment:
What about changing semantics of the return value rather than adding parameter? Return -1 in case of parse failure, 0 on BIO write error.

Also the name should be `asn1_time_print_ex` as the function is internal.

Reply:
Good thought that changing the return value is more direct. 
Done; also made the function name lower-case.